docs/site/src/components/ApiMenu.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+import React, { PureComponent, PropTypes } from 'react';
+import { camelCase } from 'docs/site/src/utils/helpers';
```

Comment:
What about sorting the import by scope? That mean, this one at the end of the list.